store/json.go

```diff
@@ -71,8 +71,18 @@ func (kvs *jsonFileStore) Read(key string, value interface{}) error {
 		}
 		defer file.Close()
 
+		b, err := ioutil.ReadAll(file)
+		if err != nil {
+			return err
+		}
+
+		if len(b) == 0 {
+			log.Printf("Unable to read file %s, was empty", kvs.fileName)
+			return nil
```

Comment:
i would suggest return ErrEmptyFile and handle this in restore function (network/manager.go and ipam/manager.go) as like ErrKeyNotFound

Reply:
@tamilmani1989 , this store file is also read by CNS to save CNS state, if are returning error then we need to handle it in there as well.